integrated_channels/canvas/client.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+"""
+Client for connecting to Canvas.
+"""
+
+from django.apps import apps
+
+from integrated_channels.integrated_channel.client import IntegratedChannelApiClient
+
+
+class CanvasAPIClient(IntegratedChannelApiClient):
+    """
+    Client for connecting to Canvas.
+
+    Code to obtain access tokens and posting to the courses and
+    completion status endpoints.
+    """
+
+    def __init__(self, enterprise_configuration):
+        """
+        Instantiate a new client.
+
+        Args:
+            enterprise_configuration (CanvasEnterpriseCustomerConfiguration): An enterprise customers's
+            configuration model for connecting with Canvas
+        """
+        super(CanvasAPIClient, self).__init__(enterprise_configuration)
+        self.global_canvas_config = apps.get_model('canvas', 'CanvasGlobalConfiguration').current()
+        self.session = None
+        self.expires_at = None
+
+    def create_course_completion(self, user_id, payload):  # pylint: disable=unused-argument
```

Comment:
I do not expect this Pr to implement the client, so just creating a blank impl enough to give us the skeleton to work off of